cmd/poller/exporter/exporter.go

```diff
@@ -72,6 +72,14 @@ func New(c, n string, o *options.Options, p conf.Exporter, params *conf.Poller)
 	}
 	if params != nil {
 		abc.Metadata.SetGlobalLabel("datacenter", params.Datacenter)
+		labels := params.Labels
+		if labels != nil {
+			for _, labelPtr := range *labels {
```

Comment:
would it make sense to replace this for loop with
`abc.Metadata.SetGlobalLabels(labelPtr)`